.babelrc

```diff
@@ -1,4 +1,3 @@
 {
-  "presets": ["es2015", "react", "stage-0"],
-  "plugins": ["transform-react-constant-elements", "transform-react-inline-elements"]
```

Comment:
these plugins should only be used in production, moved into prod webpack babel query